modules/rostests/apitests/atl/devenv/ATLTest.sln

```diff
@@ -11,6 +11,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "CSimpleMap", "CSimpleMap.vc
 EndProject
```

Comment:
Hmm is it needed to keep the VS solutions? Aren't they generated by CMake if you choose to "configure VSSolution" ?

Reply:
Yes they are needed because how else are we going to test against the MS ATL?